cit3.0-web/src/components/MapContainer/MapContainer.test.js

```diff
@@ -0,0 +1,68 @@
+import ReactDOM from 'react-dom';
+import {render, fireEvent, cleanup, waitFor } from '@testing-library/react'
+import App from '../../App';
+import MapContainer from './MapContainer';
+import axiosMock from 'axios';
+import axios from 'axios';
+
+
+afterEach(cleanup)
+
+const address = '501 Belleville St, Victoria';
+const changeEvent = (content) => {
+    return {
+        target: {
+        value: content
+        }
+    }
+}
+
+describe("MapContainer", () => {
+
+    describe("Layout", () => {
+        it('renders a map',() => {
+        const { getByText} = render(<MapContainer />)
+        expect(getByText(/leaflet/i).textContent).toBe("Leaflet")
```

Comment:
Small indentation change here
```suggestion
            const { getByText} = render(<MapContainer />)
            expect(getByText(/leaflet/i).textContent).toBe("Leaflet")
```